core/scratch_block_comment.js

```diff
@@ -58,17 +58,27 @@ Blockly.ScratchBlockComment = function(block, text, id, x, y, minimized) {
    */
   this.text_ = text;
   /**
-   * The x position of this comment in workspace coordinates.
+   * Whether this comment needs to be auto-positioned (based on provided values
+   * for x and y position).
+   * @type {boolean}
+   * @private
+   */
+  this.needsAutoPositioning_ = !x && x != 0 && !y && y != 0;
```

Comment:
Is there a value in particular that causes this? Is it always null or NaN? this is a pretty hard to parse line, could be simplified if you knew it was always null/NaN and just checked for that. But not sure if there is really a better way.

Reply:
I want the flag to be set if both x and y are any of: NaN, null, or undefined